specs-explained/index.html

```diff
@@ -143,67 +143,67 @@
         
     </div>
 </header>
-
-    
-    <div id="fh5co-home" class="small-home centre">
-        
-
-          
-          <h1 class="singleHeader">Specifications</h1>
-
-   
-    <div class="container content-container" style="text-align: center;">
-
-            <style type="text/css">
-
-.content-container {
-    display: flex;
-    justify-content: space-between;
-    max-width: 1124px;
-}
-
- img {
-  max-width: 53px;
- }
-</style>
-<div class="specsBox_container">
-<div class="specsBox">
-<img src="../images/bitcoin.png">
-<p><br><br>
-Built using Bitcoin as a base, receiving any future Bitcoin <a href="https://bitcoinmagazine.com/articles/taproot-coming-what-it-and-how-it-will-benefit-bitcoin/">updates</a> upon specification release confirmation.</p>
-<p><span style="opacity: 0.4; display: block;">Vertcoin was one of the first coins to lock in and activate SegWit. It was activated May 7, 2017</span>
-<br><br>
-<br><br></p>
-</div>
-<div class="specsBox">
-<img src="../images/litecoin.png">
-<p><br><br>
-Has the same block time (2.5 mins) as Litecoin and same supply of coins (84,000,000 coins).
-<span style="opacity: 0.4; display: block;">These changes are well within scaling limits of blockchain growth and orphan generation (&gt;1:30)</span>
-<br><br></p>
-</div>
-<div class="specsBox">
-<img src="../images/ethereum.png">
-<p><br><br>
-Verthash is Vertcoin&rsquo;s mining algorithm which is similar to Ethereum&rsquo;s Ethash (Verthash), meaning GPUs mine competitively.
-<span style="opacity: 0.4; display: block;">This ensures maximum distribution of hashrate (security). It is done via IO bound memory constraints. Verthash is the most advanced mining algorithm to date.</span>
-<br><br>
-<br><br></p>
-</div>
-<div class="specsBox">
-<img class="miningalgo" src="../images/kgw.png">
-<p><br><br>
-Difficulty adjustment occurs every block (~2.5 minutes) as opposed to Bitcoin (~2 weeks).
-<span style="opacity: 0.4; display: block;">Kimoto Gravity Well is the algorithm used and is necessary as a smaller hashrate coin.</span>
-<br><br>
-<br><br></p>
-</div>
-</div>
-
-
-    </div>
-    
-    <div class="singleFooter">
+
+    
+    <div id="fh5co-home" class="small-home centre">
+        
+
+          
+          <h1 class="singleHeader">Specifications</h1>
+
+   
+    <div class="container content-container" style="text-align: center;">
+
+            <style type="text/css">
+
+.content-container {
+    display: flex;
+    justify-content: space-between;
+    max-width: 1124px;
+}
+
+ img {
+  max-width: 53px;
+ }
+</style>
+<div class="specsBox_container">
+<div class="specsBox">
+<img src="../images/bitcoin.png">
+<p><br><br>
+Built using Bitcoin as a base, receiving any future Bitcoin <a href="https://bitcoinmagazine.com/articles/taproot-coming-what-it-and-how-it-will-benefit-bitcoin/">updates</a> upon specification release confirmation.</p>

```

Comment:
I think we should remove the BitcoinMagazine link. It's not authoritative for one and it's also old information now concerning Taproot activation.